src/main/java/com/artipie/npm/proxy/http/RsNotFound.java

```diff
@@ -41,8 +42,10 @@
     public CompletionStage<Void> send(final Connection connection) {
         return connection.accept(
             RsStatus.NOT_FOUND,
-            new ListOf<Map.Entry<String, String>>(
-                new MapEntry<>("Content-Type", "application/json")
+            new Headers.From(
```

Comment:
@brastak is this related to LoggingSlice? What does it add?

Reply:
@HDouss it's related with upgrade to http 0.8. Signature was changed